hw/src/scss/_variables.scss

```diff
@@ -0,0 +1,2 @@
+// Для body (BODY)
+$BGFORBODY: url(../img/background.png);
```

Comment:
It is not considered a best practice to directly style the "body" tag. I would suggest putting a wrapper inside the body (`<div class="wrapper"></div>`), and adding styles to it like so:
```
width: 100vw;
height: 100vh;
background-size: cover;
background: ...
```